python_telnet_vlc/vlctelnet.py

```diff
@@ -274,7 +280,8 @@ def info(self):
     # Skipping stats
     def get_time(self):
         """Seconds elapsed since stream's beginning."""
-        return int(self.run_command('get_time')[0])
+        stream_time = self.run_command('get_time')[0]
+        return int(stream_time) if stream_time else 0
```

Comment:
Is the user of the library responsible for not calling these methods unless there's a stream playing? In that case I can remove these guards.

I'd appreciate if we can still add the debug logging.

Reply:
I like the addition, thank you!

Reply:
Thanks!